.dockerfile

```
@@ -0,0 +1,14 @@
+FROM python:3.10.0
+
+LABEL maintainer="Gorgeous-Ghouls"
+
+RUN pip install poetry
+
+RUN git clone https://github.com/Gorgeous-Ghouls/Blak.git /home/Blak
+
+WORKDIR /home/Blak/server
+RUN poetry install
```

```suggestion
ENV PYTHONPATH=${PYTHONPATH}:${PWD}
ENV PATH="/root/.local/bin:${PATH}"

RUN poetry config virtualenvs.create false
RUN poetry install --no-dev
```
no need to make a venv as we already in docker
same with dev deps